lib/parser/prism/offset_cache.rb

```diff
@@ -0,0 +1,11 @@
+class OffsetCache
```

Comment:
In principle I agree with your choice to make this into its own object for encapsulation, but I'm seeing the method calls to `char_offset` slow it down significantly enough that I think we should expose the hash itself as the `offset_cache` instead.

Reply:
Can we remove this file?

Reply:
blah, 🤦 thanks